app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+import datetime

```

Comment:
fix imports here, according to the best practicies 
https://peps.python.org/pep-0008/#imports:~:text=Imports%20should%20be,library%20specific%20imports.